client/lib/site/utils.js

```diff
@@ -122,5 +122,25 @@ export default {
 		}
 
 		return true;
+	},
+
+	isMainNetworkSite( site ) {
+		if ( ! site ) {
+			return false;
+		}
+
+		if ( site.options && site.options.is_multi_network ) {
+			return false;
+		}
+
+		if ( typeof site.is_multisite === 'boolean' ) {
```

Comment:
I think it will be clearer if we do `if ( ! site.is_multisite ) { return false } and then on a non nested `if` we do the mapped check, what do you think?